### PR TITLE
rewrite pc extractor

### PR DESCRIPTION
**Pull request by livnev**

### CODE REVIEW

**Comment by livnev** [comment]
@MrChico ready for review 😊

File: examples/multipleInternals/src/lemmas.k.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
+```k
```
Comment (MrChico) [inline examples/multipleInternals/src/lemmas.k.md]: These are already present in rules.k

**Comment by MrChico** [comment]
Remove lemmas.k and I'm happy

File: examples/multipleInternals/src/lemmas.k.md (hunk 1 above)
Comment (livnev) [reply examples/multipleInternals/src/lemmas.k.md]: Cool, I will also remove them from `multipleCalls` (where they were copy-pasted from).

**Pull request merged**